test/e2e/generate_systemd_test.go

```diff
@@ -107,7 +107,7 @@ var _ = Describe("Podman generate systemd", func() {
 		session = podmanTest.Podman([]string{"generate", "systemd", "--time", "1234", "foobar"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
-		Expect(session.OutputToString()).To(ContainSubstring("TimeoutStopSec=1294"))
+		Expect(session.OutputToString()).To(ContainSubstring("TimeoutStopSec=1234"))
```

Comment:
To safe some time this should be changed back.

Reply:
You're an excellent CI bot :) Thank you!